content/applications/inventory_and_mrp/purchase/manage_deals/blanket_orders.rst

```diff
@@ -0,0 +1,203 @@
+:guilabel:`New`. This creates (and navigates to) a new :guilabel:`Blanket Order form`.
+
+From this new :guilabel:`Blanket Order form`, different fields and settings can be set and
+configured, so there are pre-determined rules that the recurring long-term agreement must follow:
+
+- :guilabel:`Purchase Representative`: is the user assigned to this specific blanket order. By
+  default, this is the user who created the agreement; the user can be changed directly from the
+  drop-down menu next to this field.
+- :guilabel:`Agreement Type`: is the type of purchase agreement this blanket order is classified
+  as. In Odoo, blanket orders are the only official purchase agreement.
+- :guilabel:`Vendor`: is the supplier to whom this agreement is tied, either once or on a recurring
+  basis. The vendor can be selected directly from the drop-down menu next to this field.
+- :guilabel:`Currency`: is the agreed-upon currency that will be used for this exchange. If
+  multiple currencies have been activated in the database, the currency can be changed from the
+  drop-down menu next to this field.
+- :guilabel:`Agreement Deadline`: is the date that this purchase agreement will be set to expire on
+  (if desired). If this blanket order should not expire, leave this field blank.
+- :guilabel:`Ordering Date`: is the date that this blanket order should be placed on if a new
+  quotation is created directly from the blanket order form. If a new quotation is created, this
+  value will automatically populate the *Order Deadline* field on the Request for Quotation (RFQ).
+- :guilabel:`Delivery Date`: is the expected delivery date that the products included in an
+  :abbr:`RFQ (Request for Quotation)` created directly from the blanket order form will be expected
+  to arrive. If a new quotation is created, this value will automatically populate the
+  *Expected Arrival* field on the :abbr:`RFQ (Request for Quotation)`.
+- :guilabel:`Source Document`: is the source Purchase Order (PO) that this blanket order will be
+  tied to. If this blanket order should not be tied to any existing :abbr:`POs (Purchase Orders)`,
+  leave this field blank.
+- :guilabel:`Company`: is the company assigned to this specific blanket order. By default, this is
+  the company that the user creating the blanket order is listed under. If the database is not a
+  multi-company database, this field can not be changed, and will default to the only company
+  listed in the database.
+
+.. image:: blanket_orders/blanket-orders-new-agreement.png
+   :align: center
+   :alt: New blanket order purchase agreement with added products.
+
+Once all relevant fields have been filled out, click :guilabel:`Add a line` to add products to the
+product lines under the :guilabel:`Product` column. Then, change the quantity of each product in
+the :guilabel:`Quantity` column (if desired), and set a price in the :guilabel:`Unit Price` column.
```

Comment:
While it's correct, I think you can just describe this kind of possibilities, as it is quite obvious:
```suggestion
Once all relevant fields have been filled out, click :guilabel:`Add a line` to add products,
then, set the quantity
and unit price.
```